packages/docusaurus-theme-common/src/utils/ThemeClassnames.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export const ThemeClassnames = {
```

Comment:
```suggestion
export const ThemeClassNames = {
```

I think camelcase is more common for this wording?

Reply:
Woops, sorry my bad!